src/renderers/dom/client/utils/setInnerHTML.js

```diff
@@ -28,7 +31,20 @@ var createMicrosoftUnsafeLocalFunction = require('createMicrosoftUnsafeLocalFunc
  */
 var setInnerHTML = createMicrosoftUnsafeLocalFunction(
   function(node, html) {
-    node.innerHTML = html;
+    if ('innerHTML' in node) {
```

Comment:
I don't believe SVG elements work with an incorrect namespace, and the parser should assign it automatically in most cases.